.php_cs.dist

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
```

Comment:
Replace with:

$finder = PhpCsFixer\Finder::create()
    ->in(\_\_DIR\_\_)
   ->name('*.php')
    ->exclude('vendor')
    ->exclude('var')
;